### DISCUSSION TITLE
Disable Catch All With Directories

### DISCUSSION THREAD

**Discussion by di-org** [simple-login/app#1726] [Feature Requests]
In the event of a spam attack on directories, where an attacker generates new aliases with each message, having a "disable catch all" function, similar to domains, would be helpful to at least pause the attack.

Thanks.

**Comment by tomfreediver** [comment]
I had the same concerns as @di-org.

Disabling catch-all via toggle - like it's already implemented for subdomains - would be a useful addition to match the subdomain functionality.   And once disabled, users would explicitely create an alias for a directory (or temporarily disable catch-all) just like for subdomains. That would make such an attack vector impossible.

Has such an attack ever been documented somewhere?
